README.md

```diff
@@ -3,6 +3,8 @@
 ![lint-build-test](https://github.com/compasjs/compas/workflows/lint-build-test/badge.svg)
 [![codecov](https://codecov.io/gh/compasjs/compas/branch/main/graph/badge.svg?token=81D84CV04U)](https://codecov.io/gh/compasjs/compas)
 
+![Compas logo](docs/public/banner.svg?raw=true)
```

Comment:
We have a README sync that will generate README's for the specific packages based on this README. But these packages are published without the image, so it will show a broken link on for example NPM.

I think the cleanest would be to move this line to the top and replace `# @compas/\*`. This way the banner shows up at the top when opening the repository, but is ignored (for now (?)) when publishing the packages.

Reply:
Ah good to know! I think a better solution would be to use an url to the svg in the compas repo. So it will also work at other places.

Reply:
My newest commit should fix this.